packages/channels/botpress-channel-messenger/src/index.js

```diff
@@ -69,7 +81,11 @@ module.exports = {
     appSecret: { type: 'string', required: true, default: '', env: 'MESSENGER_APP_SECRET' },
     verifyToken: { type: 'string', required: false, default: '' },
     enabled: { type: 'bool', required: true, default: true },
+    enableProfileFields: { type: 'any', required: false, default: [], validation: v => _.isArray(v) },
```

Comment:
`enableProfileFields` name assumes the value should be boolean but it's array. I think it should be called something like `profileFields`.

Reply:
If you want to rename it, I suggest `extraProfileFields` or `optionalProfileFields`, since `profileFields` would imply that these are all the fields there are

Let's pick the final version and I'll roll an update

Reply:
`extraProfileFields` ooks good to me

Reply:
@epaminond great, will implement in my next commit